diagnostic.md

```diff
@@ -6,21 +6,25 @@ Record your responses inside the fenced code blocks below each question.
     and what are the main task(s) you perform inside an Ember Route?
 
     ```md
-    <!-- your response here -->
+    <!-- the ember application router handles all of the connections between
+    data, and models and the templates they're rendered in.
+    the main tasks in the ember route is defining those connections, building
+    the associated paths between those ember layers. 
+     -->
     ```
 
 1.  What is the command to generate a route named `boston` nested under
     `campus`?
 
     ```md
-    <!-- your response here -->
+    <!-- ember g route campus/boston -->
     ```
 
 1.  Suppose you have a nested route at the URL `/campus/boston`. How would you
     use the `link-to` helper to generate an appropriate link?
 
     ```md
-    <!-- your response here -->
+    <!-- {{#link-to 'boston'}}Boston Campus{{/link-to}} -->
```

Comment:
This would actually need to be `{{#link-to 'campus.boston'}}Boston Campus{{/link-to}}` to access that nested route.